app/helpers/application_helper.rb

```diff
@@ -472,9 +472,8 @@ def user_set?(field)
   end
 
   def notifications
-    content_tag :div, id: 'toast-notifications-container',
-                      'data-notifications': toast_notifications_data.to_json.html_safe do
-      react_component('ToastNotifications')
+    content_tag :div, id: 'toast-notifications-container' do
```

Comment:
can this wrapper be removed?

Reply:
removed it and the unused test that was using it in `test/integration_test_helper.rb`